temporalio/converter.py

```diff
@@ -388,6 +388,16 @@ def from_payload(
 class JSONProtoPayloadConverter(EncodingPayloadConverter):
     """Converter for 'json/protobuf' payloads supporting protobuf Message values."""
 
+    def __init__(self, ignore_unknown_fields: Optional[bool] = False):
```

Comment:
```suggestion
    def __init__(self, ignore_unknown_fields bool = False):
```

(can run `poe lint-types` to confirm types or just `poe lint` to confirm all is well in general)